lib/import/germline/genotype.rb

```diff
@@ -336,7 +340,7 @@ def add_variant_impact(impact)
       end
 
       def add_variant_class(variant)
-        if variant.is_a?(Integer) && variant >= 1 && variant <= 5
+        if variant.is_a?(Integer) && variant >= 1 && variant <= 7
```

Comment:
As this is an update to the assigning variant class in the genotype.rb file, have you checked that it won't affect any of the other importers?

Reply:
Its extending the range that's allowed to be in variantpathclass  and adheres to Zvariantpathclass. If any other importer is able to extract c4/5 pathogenicity values and map upto 7 in future that should be fine.